appveyor-linux-binary.yml

```diff
@@ -19,7 +19,7 @@ environment:
   PYTHON_HOME: "$HOME/venv3.11/bin"
   PYTHON_VERSION: '3.11'
   AWS_DEFAULT_REGION: us-east-1
-  NODE_VERSION: "14.17.6"
+  NODE_VERSION: "18.18.2"
```

Comment:
Does this affect other tests? And do we need to change settings for windows as well?

Reply:
The tests are passing on Windows (ref: https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli-canary-windows-dev/builds/48658868/job/ql8agf8u9fgncwtj?fullLog=true), though it might be worth it to pin `npm` on Windows too.